### PR TITLE
Update FAQ.md

### PR DESCRIPTION
**Pull request by YuriKravetc**
Adding explanation "Why do same metrics have differences in VictoriaMetrics and Prometheus dashboards?"

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -314,3 +314,10 @@ Please see [these docs](https://docs.victoriametrics.com/vmctl.html#migrating-da
 ## How to migrate data from Graphite to VictoriaMetrics?
 
 Please use [whisper-to-graphite](https://github.com/bzed/whisper-to-graphite) tool for reading the data from Graphite and pushing it to VictoriaMetrics via [Graphite import API](https://docs.victoriametrics.com/#how-to-send-data-from-graphite-compatible-agents-such-as-statsd).
+
+
+## Why do same metrics have differences in VictoriaMetrics and Prometheus dashboards?
```
Comment (hagen1778) [inline docs/FAQ.md:319]: The explanation is not full. There are two aspects to consider:
1. Difference in stored raw time series. Due to different compressions algorithms, VM may reduce precision for float values with more than 12 significant decimal digits. Please, read more about this here https://valyala.medium.com/evaluating-performance-and-correctness-victoriametrics-response-e27315627e87
2. Difference in query responses. Please read more here https://medium.com/@romanhavronenko/victoriametrics-promql-compliance-d4318203f51e

**Review by hagen1778 — CHANGES_REQUESTED** [review]
Please, see my comment

File: docs/FAQ.md [hunk 2]
```diff
@@ -314,3 +314,11 @@ Please see [these docs](https://docs.victoriametrics.com/vmctl.html#migrating-da
...
+
+There is a difference in stored raw time series. Due to different compression algorithms, VM may reduce precision for float values with more than 12 significant decimal digits. Please see [this article](https://valyala.medium.com/evaluating-performance-and-correctness-victoriametrics-response-e27315627e87)
```
Comment (hagen1778) [inline docs/FAQ.md:321]:
```suggestion
There could be a slight difference in stored values for time series. Due to different compression algorithms, VM may reduce precision for float values with more than 12 significant decimal digits. Please see [this article](https://valyala.medium.com/evaluating-performance-and-correctness-victoriametrics-response-e27315627e87)
```

File: docs/FAQ.md [hunk 3]
```diff
@@ -314,3 +314,11 @@ Please see [these docs](https://docs.victoriametrics.com/vmctl.html#migrating-da
...
+
+There is a slight difference in logic and query responses. Please see [this article](https://medium.com/@romanhavronenko/victoriametrics-promql-compliance-d4318203f51e)
```
Comment (hagen1778) [inline docs/FAQ.md:323]:
```suggestion
The query engine may behave differently for some functions. Please see [this article](https://medium.com/@romanhavronenko/victoriametrics-promql-compliance-d4318203f51e)
```

**Review by hagen1778 — APPROVED** [review]

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
Thanks!

**Pull request merged**